src/app/tests/suites/certification/ci-pics-values

```diff
@@ -759,7 +759,7 @@ LWM.S.C01.Tx=1
 PIXIT.LWM.MODE_CHANGE_FAIL=1
 PIXIT.LWM.MODE_CHANGE_OK=1
 
-LWM.S.M.CAN_TEST_MODE_FAILURE=1
+LWM.S.M.CAN_TEST_MODE_FAILURE=0
```

Comment:
if we turn off this in CI, how can we ensure that the test works?

I am conflicted here: on one hand an automated test is better than manual, however on the other side I wonder if this will bitrot. Are there any plans/issues to support this in sample apps? Or would we just accept that if devices fail they will turn off the PICS (I find that likely as it is easier to change an enabling and get certified instead of trying to fix things that the SDK does not test to be correctly implemented in CI)

Reply:
Honestly, there's absolutely nothing to enforce that someone who did support this functionality turns the pics on in the first place. I'd put money that no one has gone through cert with this PICS on and this is as fully untested in this way as it was before. At least this way we're not pretending.

We don't have any way to know from the device whether this error is likely to be returned. We can file a follow up issue for the team to reconsider the test, but for now, I'd rather have the CI PICS state what the CI actually supports. It does not support this.